spec/models/invoice_item_spec.rb

```diff
@@ -2,11 +2,11 @@
 
 RSpec.describe InvoiceItem, type: :model do
   describe 'relationships' do
-    it { should have_many :items}
-    it { should have_many :invoices}
-  end 
+    it {should belong_to(:item)}
+    it {should belong_to(:invoice)}
+  end
 
   describe 'validations' do
-    it { should define_enum_for(:status).with([:packaged, :pending, :shipped]) }
+    it { should define_enum_for(:status).with_values([:packaged, :pending, :shipped]) }
```

Comment:
This test is failing, I will pull down and try to get to pass before we merge to main